PROProtocol/PokemonStats.cs

```diff
@@ -80,5 +80,26 @@ public bool HasOnly(StatType type)
             }
             return true;
         }
+
+        public int GetEffortValue(StatType type)

```

Comment:
Is there any major difference with the already existing `GetStat()` method?
I don't think a copy/paste is required.